src/utils/Symbol_observable.js

```diff
@@ -0,0 +1,18 @@
+let $$observable
+
+if (typeof Symbol === 'function') {
+  if (!Symbol.observable) {
+    if (typeof Symbol.for === 'function') {
+      $$observable = Symbol.for('observable')
+    } else {
+      $$observable = Symbol()
+    }
+    Symbol.observable = $$observable
+  } else {
+    $$observable = Symbol.observable
+  }
+} else {
+  $$observable = '@@observable'
+}
+
+export default $$observable
```

Comment:
I'm 50/50 on this, personally. On one hand, it's not a lot of code, and there's the ol' left-pad issue. On the other hand, I tend to agree it could be centralized somewhere. I know that @sindresorhus has a ponyfill for this, but I've seen a couple of bugs in it and I haven't had the time to reach out to collaborate with him (I don't actually know him at all). Perhaps later this afternoon I'll put in a PR on his repository and if that doesn't fly I can always publish something elsewhere, ReactiveX perhaps.